cohort_4/playbook/ACTIVITIES.md

```diff
@@ -104,6 +104,7 @@
     - [Announcing to the Whole Group](#announcing-to-the-whole-group)
     - [Sharing Announcements Individually](#sharing-announcements-individually)
     - [Slack Announcements](#slack-announcements)
+- [Additional Resources](#additional-resources)
```

Comment:
Oh god sorry that was the part for codehs and I deleted it. I'll fix that

On Tue, Aug 25, 2015 at 3:56 PM Max Wofford <notifications@github.com>
wrote:

> In cohort_4/playbook/ACTIVITIES.md
> <https://github.com/hackedu/hack-camp/pull/519#discussion_r37929720>:
>
> > @@ -104,6 +104,7 @@
> >      - [Announcing to the Whole Group](#announcing-to-the-whole-group)
> >      - [Sharing Announcements Individually](#sharing-announcements-individually)
> >      - [Slack Announcements](#slack-announcements)
> > +- [Additional Resources](#additional-resources)
>
> @ssun098 <https://github.com/ssun098> This is what is causing the tests
> to fail. There doesn't seem to be an Additional Resources heading.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/hack-camp/pull/519/files#r37929720>.
>